webapp/src/Controller/API/JudgehostController.php

```diff
@@ -919,7 +925,8 @@ private function addSingleJudgingRun(
         string $outputRun,
         ?string $teamMessage,
         string $metadata,
-        ?string $testcasedir
+        ?string $testcasedir,
+        ?string $compare_meta,
```

Comment:
Nit:
```suggestion
        ?string $compareMeta,
```
to be consistent with the other arguments